packages/core/src/Stores/notification-store.js

```diff
@@ -158,6 +156,10 @@ export default class NotificationStore extends BaseStore {
         return notification_content;
     }
 
+    get is_notifications_empty() {
```

Comment:
Do we need this compute?

Reply:
since we have an array called `notfication` inside store named `notification`, its looking a bit messy when we are taking the array from store to component and then accessing its length and then storing it in a variable. the below code looks clean and calm, right?
![image](https://github.com/binary-com/deriv-app/assets/122449658/af84b1cc-2cdb-4bf6-b3cb-2cc46725c42e)

Reply:
Sounds good